### DISCUSSION TITLE
Schema directory instead of schema file

### DISCUSSION THREAD

**Discussion by terion-name** [directus/directus#11262] [Feature Requests]
Schema dumps are a great tool that can be used not only as a method to transfer project through environments, but also to distribute things like modules. For example one can publish a schema that winds up a ready-to-use blog setup, or a content-design system, or a e-commerce solution. Very simple and convenient. But when this all is settled in one huge file it would be difficult to combine multiple "modules". So the idea is to separate schemas per collection and put them into a directory where one can add files and apply. Also it would be useful to support nesting to organise them as modules, like:
```
schema
|- shop
|-|- item.yml
|-|- cart.yml
```

**Comment by rijkvanzanten** [comment]
Love it. It's exactly what we talked about before @keesvanbemmel 🙂

**Reply by terion-name** [reply]
but this would play better with https://github.com/directus/directus/discussions/11249 ;)

**Reply by keesvanbemmel** [reply]
Splitting up the yamls is indeed a great way of getting rid of weird github diff comparisons. Makes schema changes across teams much easier!

**Comment by jamescammarano** [comment]
This has me very excited about the possibilities.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
